pages/account/manage/links.js

```diff
@@ -112,7 +112,7 @@ export default function ManageLinks({ BASE_URL, username, links }) {
           </Button>
 
           {linkList.length !== 0 && !reorder && (
-            <Button onClick={() => setReorder(true)}>
+            <Button onClick={() => setReorder(true)} disable={linkList.length < 2}>
```

Comment:
From the line above, I think we need to remove this so it is displayed? `linkList.length !== 0`

Reply:
`linkList.length !== 0` is for displaying/hiding reorder button. if we removed that, it will display _disabled_ reorder button even if there are no links. 

`linkList.length !== 0`will check if any links are there  and  `disable={linkList.length < 2}`will check how many and then disable/enable. So we need both.

Reply:
Yes you discussed this with our maintainer here https://github.com/EddieHubCommunity/BioDrop/issues/9221#issuecomment-1742009718

Reply:
okay and will do the changes.